src/config/scrivitoSites.ts

```diff
@@ -1,7 +1,16 @@
-import { Obj, currentSiteId, getInstanceId, load, navigateTo } from 'scrivito'
+import {
+  Link,
+  Obj,
+  currentSiteId,
+  getInstanceId,
+  load,
+  navigateTo,
+  urlFor,
+} from 'scrivito'
 import { isMultitenancyEnabled } from './scrivitoTenants'
 
 const location = typeof window !== 'undefined' ? window.location : undefined
+const rootContentId = import.meta.env.SCRIVITO_ROOT_CONTENT_ID
```

Comment:
High level feedback: I would prefer, that there is _always_ a `rootContentId`. Since we want to run this in a multi-tenancy fashion custom env variables is not an option (or only an option if it applies to _all_ tenants).

Reply:
Is this requirement still conditional depending on `isMultitenancyEnabled`?
What if the Portal App is used as a baseline for a new project without a multi-tenant requirement?

Reply:
> Is this requirement still conditional depending on `isMultitenancyEnabled`?

No. I think these are two independent things. The rootContentId is used to ensure that in _one_ tenant you can have multiple sites where some sites are _not_ rendered by the scrivito portal app. `isMultitenancyEnabled` is to tell the scrivito portal app, that it can render _multiple_ tenants.

> What if the Portal App is used as a baseline for a new project without a multi-tenant requirement?

In that case they would start out with the initial content, that contains the rootContentId. If they have other requirements in the project they are free to adjust the code.